qtox.pro

```diff
@@ -272,135 +230,133 @@ unix:!macx:!android {
+    src/widget/genericchatroomwidget.h \
+    src/widget/form/genericchatform.h \
+    src/widget/tool/adjustingscrollarea.h \
+    src/widget/form/loadhistorydialog.h \
+    src/widget/form/setpassworddialog.h \
+    src/widget/form/tabcompleter.h \
+    src/net/autoupdate.h \
+    src/widget/tool/callconfirmwidget.h \
+    src/widget/systemtrayicon.h \
+    src/widget/qrwidget.h \
+    src/widget/systemtrayicon_private.h \
+    src/widget/loginscreen.h \
+    src/ipc.h
 
-    SOURCES += \
-        src/widget/form/addfriendform.cpp \
-        src/widget/form/settingswidget.cpp \
-        src/widget/form/settings/generalform.cpp \
-        src/widget/form/settings/privacyform.cpp \
-        src/widget/form/settings/avform.cpp \
-        src/widget/form/profileform.cpp \
-        src/widget/form/filesform.cpp \
-        src/widget/tool/chattextedit.cpp \
-        src/widget/tool/friendrequestdialog.cpp \
-        src/widget/widget.cpp \
-        src/video/netcamview.cpp \
-        src/widget/friendlistwidget.cpp \
-        src/widget/tool/adjustingscrollarea.cpp \
-        src/widget/form/loadhistorydialog.cpp \
-        src/widget/form/setpassworddialog.cpp \
-        src/widget/form/tabcompleter.cpp \
-        src/widget/flowlayout.cpp \
-        src/net/autoupdate.cpp \
-        src/widget/tool/callconfirmwidget.cpp \
-        src/widget/systemtrayicon.cpp \
-        src/widget/groupwidget.cpp \
-        src/widget/friendwidget.cpp \
-        src/widget/form/chatform.cpp \
-        src/widget/form/groupchatform.cpp \
-        src/widget/form/genericchatform.cpp \
-        src/friend.cpp \
-        src/friendlist.cpp \
-        src/group.cpp \
-        src/grouplist.cpp \
-        src/persistence/smileypack.cpp \
-        src/widget/emoticonswidget.cpp \
-        src/widget/style.cpp \
-        src/widget/tool/croppinglabel.cpp \
-        src/widget/maskablepixmapwidget.cpp \
-        src/video/videosurface.cpp \
-        src/net/toxuri.cpp \
-        src/net/toxdns.cpp \
-        src/persistence/toxsave.cpp \
-        src/persistence/serialize.cpp \
-        src/chatlog/chatlog.cpp \
-        src/chatlog/chatline.cpp \
-        src/chatlog/chatlinecontent.cpp \
-        src/chatlog/chatlinecontentproxy.cpp \
-        src/chatlog/content/text.cpp \
-        src/chatlog/content/spinner.cpp \
-        src/chatlog/content/filetransferwidget.cpp \
-        src/chatlog/chatmessage.cpp \
-        src/chatlog/content/image.cpp \
-        src/chatlog/customtextdocument.cpp\
-        src/widget/form/settings/aboutform.cpp \
-        src/widget/form/settings/advancedform.cpp \
-        src/chatlog/content/notificationicon.cpp \
-        src/chatlog/content/timestamp.cpp \
-        src/chatlog/documentcache.cpp \
-        src/chatlog/pixmapcache.cpp \
-        src/persistence/offlinemsgengine.cpp \
-        src/widget/qrwidget.cpp \
-        src/widget/genericchatroomwidget.cpp \
-        src/widget/loginscreen.cpp \
-        src/ipc.cpp
-}
+SOURCES += \
```

Comment:
Either add a comment that here's the stuff that is not portable/useful on android, or merge it with `SOURCES` below.